pypsi/completers.py

```diff
@@ -22,6 +22,82 @@
 from pypsi.os import path_completer  # noqa
 
 
+def command_completer(parser, shell, args, prefix, case_sensitive=False):
+    '''
+    Completion function that can tab complete options,
```

Comment:
Mention here that the developer should call this function with the same arguments passed into the command's `complete()` method.